### DISCUSSION TITLE
Frigate Hass integration notifications offline only?

### DISCUSSION THREAD

**Discussion by Duskspeed** [blakeblackshear/frigate#9057] [Ask A Question] [answered]
Hey guys just got frigate, mqtt and Hass up and running and I'm really enjoying playing around with the setup(waiting for my coral), I have a question regarding device notifications using Hass integration, is it possible to have the notifications without internet access, such as a local network only push notification?

I have no need for external access or alerts so the system is blocked from accessing the internet but notifications seem to require it from what I can observe.

Thanks

**Comment by NickM-27** [comment, marked as answer]
Yes, HA supports websocket notifications which work 100% locally

**Reply by Duskspeed** [reply]
Hi Nick, absolutely fantastic, that pointed me in the right direction and I got it working, thank you so much😭 😊
